From 3f5fc0093ec9d26a78df331129f56cc849313673 Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Mon, 12 Jun 2023 20:56:19 +0200
Subject: [PATCH 141/N] CI: updates for mstorsjo/llvm-mingw moving to 20.04

add-apt-repository is no longer included by default and
needs to be installed

also disable fail-fast while at it, we care about all jobs
---
 .github/workflows/mingw.yml | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/.github/workflows/mingw.yml b/.github/workflows/mingw.yml
index d32b528..15cb167 100644
--- a/.github/workflows/mingw.yml
+++ b/.github/workflows/mingw.yml
@@ -225,6 +225,7 @@ jobs:
     container:
       image: mstorsjo/llvm-mingw:latest
     strategy:
+      fail-fast: false
       matrix:
         arch: ['x86_64', 'i686', 'aarch64', 'armv7']
     steps:
@@ -233,6 +234,8 @@ jobs:
       - name: Install deps
         run: |
           export DEBIAN_FRONTEND=noninteractive
+          apt-get update -qq
+          apt-get install -qqy software-properties-common
           add-apt-repository --yes ppa:deadsnakes/ppa
           apt-get update -qq
           apt-get install -qqy autoconf-archive python3.10-dev python3.10
@@ -274,6 +277,7 @@ jobs:
     needs: [cross-llvm-mingw]
     runs-on: windows-latest
     strategy:
+      fail-fast: false
       matrix:
         arch: ['x86_64', 'i686']
     steps:
